multichain-testing/test/fast-usdc/fast-usdc.test.ts

```diff
@@ -107,23 +110,64 @@ test.after(async t => {
   deleteTestKeys(accounts);
 });
 
+type VStorageClient = Awaited<ReturnType<typeof commonSetup>>['vstorageClient'];
+const agoricNamesQ = (vsc: VStorageClient) =>
```

Comment:
not a blocker but I find these one-off abstractions a little hard to work with. I hope we can design good general abstractions in `@agoric/client-utils` to invest in. https://github.com/Agoric/agoric-sdk/pull/10713/ attempts to replace test calls to `VStorageClient` with `SmartWalletKit`, getting typed data for query paths.

If the client-utils abstractions aren't right, let's work on to improve them, so all consumers benefit.

Reply:
Thanks for #10713; I looked it over and made some comments.

The cycle time in multi-chain testing is extremely high, so I'm not eager to go beyond this 1 test file for this feature.

In general, my habit is to experiment until I see a pattern that works well in at least 2 or 3 cases and then DRY it out.